build.properties

```diff
@@ -11,4 +11,4 @@ privilegedUser=postgres
 privilegedPassword=
 preparethreshold=5
 loglevel=0
-protocolVersion=0
+#protocolVersion=0
```

Comment:
Now Travis has done a run I see that a load of errors are being thrown. Is protocol 0 supposed to work ?